src/api.js

```diff
@@ -135,6 +135,10 @@ var api = {
     "https://firebasehosting.googleapis.com"
   ),
   cloudRunApiOrigin: utils.envOverride("CLOUD_RUN_API_URL", "https://run.googleapis.com"),
+  serviceUsageOrigin: utils.envOverride(
```

Comment:
If this is what we should be using over something else, can we mark the others as deprecated with a comment?

Reply:
It's what we're using instead of the previously inlined `servicemanagement.googleapis.com` origin. Nothing to comment out from that perspective.